### DISCUSSION TITLE
setup a CADDY_GLOBAL_OPTIONS config in docker-compose environment.

### DISCUSSION THREAD

**Discussion by lam0819** [php/frankenphp#355] [Q&A]
i want to set 

environment:
- CADDY_GLOBAL_OPTIONS="auto_https off"

but seems it cannot recognise the auto_https off.

Is there anythings i miss understand?

**Comment by Henridv** [comment]
If you don't want to use HTTPS you should set `SERVER_NAME=http://` instead.

  About `auto_https`, from the [Caddy docs](https://caddyserver.com/docs/caddyfile/options#auto-https):
> This option does not affect Caddy's default protocol, which is always HTTPS.

**Reply by danilopolani** [reply]
Thank you! I was having the same issue. Apparently there's a bit of misuderstanding (counting myself in) about `auto_https` since it's just disable auto-redirects and eventually the SSL generation, but as you said Caddy would still bind to HTTPS.
More on this. https://caddy.community/t/making-sense-of-auto-https-and-why-disabling-it-still-serves-https-instead-of-http/9761
